quickemailverification.gemspec

```
@@ -16,5 +16,5 @@ Gem::Specification.new do |gem|
   gem.files = Dir["lib/**/*"]
 
   gem.add_dependency "faraday", "~> 0.9"
-  gem.add_dependency "json", "~> 1.8"
+  gem.add_dependency "json", "~> 2.5.1"
```

```suggestion
  gem.add_dependency "json", "~> 2.5."
```

We have checked JSON version "\~> 2.5.1" with our library but we are getting below warning while building the gem for our library.
WARNING:  pessimistic dependency on json (\~> 2.5.1) may be overly strict
  if json is semantically versioned, use:
    add_runtime_dependency 'json', '\~> 2.5', '>= 2.5.1'
WARNING:  See http://guides.rubygems.org/specification-reference/ for help

Thanks